nengo_loihi/builder.py

```diff
@@ -289,21 +289,9 @@ def build_ensemble(model, ens):
         # scaled_encoders = encoders * (gain / ens.radius)[:, np.newaxis]
         scaled_encoders = encoders * gain[:, np.newaxis]
 
-    # --- encoders
-    # synapses = CxSynapses(scaled_encoders.shape[1])
-    # synapses.set_full_weights(scaled_encoders.T)
-    # group.add_synapses(synapses, name='encoders')
-
-    # --- encoders for interneurons
-    synapses2 = CxSynapses(2*scaled_encoders.shape[1], label="inter_encoders")
-    inter_scale = 1. / (model.dt * INTER_RATE * INTER_N)
-    interscaled_encoders = scaled_encoders * inter_scale
-    synapses2.set_full_weights(
-        np.vstack([interscaled_encoders.T, -interscaled_encoders.T]))
-    group.add_synapses(synapses2, name='encoders2')
-
     model.add_group(group)
 
+    model.objs[ens]['group'] = group
```

Comment:
It feels a little weird that we're storing the group 4 separate times ... is there a possibility that the `'in'` and `'out'` would be something else?

Reply:
I didn't want to assume, but you're probably right. I'll change it.